PWGHF/DataModel/CandidateReconstructionTables.h

```diff
@@ -1531,6 +1531,178 @@ DECLARE_SOA_TABLE(HfCandScMcGen, "AOD", "HFCANDSCMCGEN", //!
+                  hf_cand_dstar::D0CosThetaStar<hf_cand::PxProng0, hf_cand::PyProng0, hf_cand::PzProng0, hf_cand::PxProng1, hf_cand::PyProng1, hf_cand::PzProng1>,
+                  hf_cand_dstar::D0ImpactParameterProngSqSum<hf_cand::ImpactParameter0, hf_cand::ImpactParameter1>,
+                  /* dynamic columns that use candidate momentum components */
+                  hf_cand_dstar::D0Pt<hf_cand_dstar::D0Px, hf_cand_dstar::D0Py>,
+                  hf_cand_dstar::D0Pt2<hf_cand_dstar::D0Px, hf_cand_dstar::D0Py>,
+                  hf_cand_dstar::D0P<hf_cand_dstar::D0Px, hf_cand_dstar::D0Py, hf_cand_dstar::D0Pz>,
+                  hf_cand_dstar::D0P2<hf_cand_dstar::D0Px, hf_cand_dstar::D0Py, hf_cand_dstar::D0Pz>,
+                  hf_cand_dstar::D0PVector<hf_cand_dstar::D0Px, hf_cand_dstar::D0Py, hf_cand_dstar::D0Pz>,
+                  hf_cand_dstar::D0CPA<collision::PosX, collision::PosY, collision::PosZ, hf_cand::XSecondaryVertex, hf_cand::YSecondaryVertex, hf_cand::ZSecondaryVertex, hf_cand_dstar::D0Px, hf_cand_dstar::D0Py, hf_cand_dstar::D0Pz>,
+                  hf_cand_dstar::D0CPAXY<collision::PosX, collision::PosY, hf_cand::XSecondaryVertex, hf_cand::YSecondaryVertex, hf_cand_dstar::D0Px, hf_cand_dstar::D0Py>,
+                  hf_cand_dstar::D0Ct<collision::PosX, collision::PosY, collision::PosZ, hf_cand::XSecondaryVertex, hf_cand::YSecondaryVertex, hf_cand::ZSecondaryVertex, hf_cand_dstar::D0Px, hf_cand_dstar::D0Py, hf_cand_dstar::D0Pz>,
+                  hf_cand_dstar::D0ImpactParameterXY<collision::PosX, collision::PosY, collision::PosZ, hf_cand::XSecondaryVertex, hf_cand::YSecondaryVertex, hf_cand::ZSecondaryVertex, hf_cand_dstar::D0Px, hf_cand_dstar::D0Py, hf_cand_dstar::D0Pz>,
+                  hf_cand_dstar::D0MaxNormalisedDeltaIP<collision::PosX, collision::PosY, hf_cand::XSecondaryVertex, hf_cand::YSecondaryVertex, hf_cand::ErrorDecayLengthXY, hf_cand_dstar::D0Px, hf_cand_dstar::D0Py, hf_cand::ImpactParameter0, hf_cand::ErrorImpactParameter0, hf_cand::ImpactParameter1, hf_cand::ErrorImpactParameter1, hf_cand::PxProng0, hf_cand::PyProng0, hf_cand::PxProng1, hf_cand::PyProng1>,
+                  hf_cand_dstar::D0Eta<hf_cand_dstar::D0Px, hf_cand_dstar::D0Py, hf_cand_dstar::D0Pz>,
+                  hf_cand_dstar::D0Phi<hf_cand_dstar::D0Px, hf_cand_dstar::D0Py>,
+                  hf_cand_dstar::D0Y<hf_cand_dstar::D0Px, hf_cand_dstar::D0Py, hf_cand_dstar::D0Pz>,
+                  hf_cand_dstar::D0E<hf_cand_dstar::D0Px, hf_cand_dstar::D0Py, hf_cand_dstar::D0Pz>,
+                  hf_cand_dstar::D0E2<hf_cand_dstar::D0Px, hf_cand_dstar::D0Py, hf_cand_dstar::D0Pz>);
+
+// extended table with expression columns that can be used as arguments of dynamic columns
+DECLARE_SOA_EXTENDED_TABLE_USER(HfD0FromDstarExt, HfD0FromDstarBase, "HFD0FRMDSTREXT", hf_cand_dstar::D0Px, hf_cand_dstar::D0Py, hf_cand_dstar::D0Pz);
+using HfD0fromDstar = HfD0FromDstarExt;
+
+DECLARE_SOA_TABLE(HfCandDStarBase, "AOD", "HFDSTARCAND",
```

Comment:
```suggestion
DECLARE_SOA_TABLE(HfCandDstarBase, "AOD", "HFCANDDSTARBASE",
```
Please be consistent with the existing naming conventions.

Reply:
Not addressed

Reply:
I will do it again.